atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/DualWriteKeyValueService.java

```diff
@@ -110,6 +110,12 @@ public void delete(TableReference tableRef, Multimap<Cell, Long> keys) {
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest range) {
```

Comment:
@jboreiko `deleteRange` should be idempotent, though, so retrying the delete shouldn't be a big issue.